website/docs/d/loadbalancer.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_lb"
+sidebar_current: "docs-azurerm-datasource-load-balancer-x"
+description: |-
+  Get information about an existing Load Balancer
+
+---
+
+# Data Source: azurerm_lb
+
+Use this data source to access information about an existing Load Balancer
+
+## Example Usage
+
+```hcl
+data "azurerm_lb" "test" {
+  name                = "example-lb"
+  resource_group_name = "example-resources"
+}
+
+output "loadbalancer_id" {
+  value = "${data.azurerm_lb.test.id}"
+}
+```
+
+## Argument Reference
+
+* `name` - (Required) Specifies the name of the Load Balancer.
+
+* `resource_group_name` - (Required) The name of the Resource Group in which the Load Balancer exists.
+
+## Attributes Reference
+
+The following attributes are exported:
```

Comment:
It looks like `private_ip_address` and `private_ip_addresses` properties are missing?